cmd/sonarExecuteScan.go

```diff
@@ -82,8 +85,27 @@ func sonarExecuteScan(config sonarExecuteScanOptions, _ *telemetry.CustomData, i
 	downloadClient.SetOptions(piperhttp.ClientOptions{TransportTimeout: 20 * time.Second})
 	// client for talking to the SonarQube API
 	apiClient := &piperhttp.Client{}
-	//TODO: implement certificate handling
-	apiClient.SetOptions(piperhttp.ClientOptions{TransportSkipVerification: true})
+	proxy := config.Proxy
+	if proxy != "" {
```

Comment:
Is `proxy != ""`  the same as `len(proxy) >0` ?
If proxy can be null, maybe it is better to use `len(proxy) >0`.

Reply:
I think both of them are identical as mentioned [here](https://stackoverflow.com/questions/18594330/what-is-the-best-way-to-test-for-an-empty-string-in-go).